lib/ui/mail_view_indv.dart

```diff
@@ -29,36 +76,59 @@ class MailPieceViewWidget extends StatelessWidget{
+                  fontWeight: FontWeight.bold,
+                  color: Color.fromRGBO(51, 51, 102, 1.0)),
+              ),
+              mailImage, //load link to photo
+              Container(
+              padding: EdgeInsets.all(15),
+              child:
+                Align(
+                  alignment: Alignment.topLeft,
+                  child:
+                  Wrap(
+                    direction: Axis.vertical,
+                    alignment: WrapAlignment.start,
+                    spacing: 15,
+                    children: [
+                      Text('RECEIVED: ' + DateFormat('yyyy/MM/dd').format(mailPiece.timestamp) + ' ' + DateFormat('EEE hh:mm a').format(mailPiece.timestamp),
+                          style: TextStyle(fontSize: 15)),
+                      Text('RELEVANT TEXT: \n' + mailPiece.imageText,
+                          style: TextStyle(fontSize: 15,
+                            color: Color.fromRGBO(51, 51, 102, 1.0) )),
+                    ]
+                  ),
+                ),
+              ),
+              Container(
                 padding: EdgeInsets.all(15),
+                  decoration: BoxDecoration(
+                    border: Border.all(color: Colors.black),
+                    borderRadius: new BorderRadius.circular(16.0),
+                    color: _buttonColor
+                  ),
                 child:
-                    Align(
-                      alignment: Alignment.topLeft,
-                      child:
-                      Wrap(
-                        direction: Axis.vertical,
-                        alignment: WrapAlignment.start,
-                        spacing: 15,
-                        children: [
-                          Text('RECEIVED: ' + DateFormat('yyyy/MM/dd').format(mailPiece.timestamp) + ' ' + DateFormat('EEE hh:mm a').format(mailPiece.timestamp),
-                              style: TextStyle(fontSize: 15)),
-                          Text('SENT BY: ' + mailPiece.sender,
-                              style: TextStyle(fontSize: 20,
-                                              fontWeight: FontWeight.bold,
-                                              color: Color.fromRGBO(51, 51, 102, 1.0)),
+                    Column(
+                      children: [
+                          Text('area for more actions\n\n',
+                          style: TextStyle(fontSize: 20,
+                          fontWeight: FontWeight.bold,
+                          color: Colors.white),
                           ),
-                          Text('RELEVANT TEXT: \n' + mailPiece.imageText,
-                              style: TextStyle(fontSize: 15,
-                                color: Color.fromRGBO(51, 51, 102, 1.0) ))
-                        ]
-                      ),
-                    ),
-                ),
-            ],
+                          Text('area for do more with your mail\n\n',
```

Comment:
The removed text was more information than this. We want to display the imageText here I thought, but perhaps run it though some method that formats it better at some point.

Reply:
I'm not sure what this refers to, I did remove some junk fields I was using to make sure I was obtaining the correct email.  Started with displaying the subject of the email, when that worked, I moved on to making sure the email picture displayed.

Reply:
Sorry I think I made this comment before realizing lawrence's pr wasn't in Developer. It is now and he had this area displaying the imageText from the OCR translation.